diff --git a/node_modules/hercai/hercai.js b/node_modules/hercai/hercai.js
index 9d13350..3da16c4 100644
--- a/node_modules/hercai/hercai.js
+++ b/node_modules/hercai/hercai.js
@@ -1,71 +1,71 @@
 "use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
-const axios = require('axios');
-const baseurl = "https://hercai.onrender.com/v2/hercai";
 
-
-/**
- * @typedef {Class} Hercai
- * @see {Hercai}
- * @param {Class} Hercai
- * @example const { Hercai } = require("hercai");
- * @example import { Hercai } from "hercai";
- * @type {Class}
- * @class
- */
 class Hercai {
-    constructor() {
-    };
+    constructor() {}
 
-/**
-* The Question You Want to Ask Artificial Intelligence.
-* @param {string} model "v3-beta" (GPT-4)
-* @param {string} content The Question You Want to Ask Artificial Intelligence.
-* @example client.question({model:"v3-beta",content:"how are you?"})
-* @type {string} The Question You Want to Ask Artificial Intelligence.
-* @returns {Hercai}
-* @async
-*/
-async question({model = "v3-beta",content}){
-if(!["v3"].some(ind => model == ind)) model = "v3-beta";
-if(!content || content == undefined || content == null)throw new Error("Please specify a question!");
-try{
-var api = await axios.get(`https://hercai.onrender.com/${model}/hercai?question=`+encodeURI(content),{
-    headers: {
-        "content-type": "application/json",
-    },
-})
-return api.data;
-}catch(err){
-throw new Error("Error: "+ err.message)   
-}
-}
+    /**
+     * The Question You Want to Ask Artificial Intelligence.
+     * @param {string} model "v3-beta" (GPT-4)
+     * @param {string} content The Question You Want to Ask Artificial Intelligence.
+     * @example client.question({model:"v3-beta",content:"how are you?"})
+     * @type {string} The Question You Want to Ask Artificial Intelligence.
+     * @returns {Hercai}
+     * @async
+     */
+    async question({ model = "v3-beta", content }) {
+        if (!["v3"].some(ind => model == ind)) model = "v3-beta";
+        if (!content || content == undefined || content == null) throw new Error("Please specify a question!");
+        try {
+            const apiUrl = `https://hercai.onrender.com/${model}/hercai?question=` + encodeURI(content);
+            const response = await fetch(apiUrl, {
+                method: 'GET',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+            });
 
-/**
-* Tell Artificial Intelligence What You Want to Draw.
-* @param {string} model "v1" , "v2" , "v2-beta" , "v3" (DALL-E) , "lexica" , "prodia"
-* @param {string} prompt Tell Artificial Intelligence What You Want to Draw.
-* @example client.drawImage({model:"v1",prompt:"anime girl"})
-* @type {string} Tell Artificial Intelligence What You Want to Draw.
-* @returns {Hercai}
-* @async
-*/
-async drawImage({model = "v3",prompt}){
-    if(!["v1","v2","v2-beta","v3","lexica","prodia"].some(ind => model == ind)) model = "prodia";
-    if(!prompt || prompt == undefined || prompt == null)throw new Error("Please specify a prompt!");
-    try{
-    var api = await axios.get(`https://hercai.onrender.com/${model}/text2image`+"?prompt="+encodeURI(prompt),{
-        headers: {
-            "content-type": "application/json",
-        },
-    })
-    return api.data;
-    }catch(err){
-    throw new Error("Error: "+ err.message)   
-    }
+            if (!response.ok) {
+                throw new Error(`HTTP error! Status: ${response.status}`);
+            }
+
+            const data = await response.json();
+            return data;
+        } catch (err) {
+            throw new Error("Error: " + err.message);
+        }
     }
 
+    /**
+     * Tell Artificial Intelligence What You Want to Draw.
+     * @param {string} model "v1" , "v2" , "v2-beta" , "v3" (DALL-E) , "lexica" , "prodia"
+     * @param {string} prompt Tell Artificial Intelligence What You Want to Draw.
+     * @example client.drawImage({model:"v1",prompt:"anime girl"})
+     * @type {string} Tell Artificial Intelligence What You Want to Draw.
+     * @returns {Hercai}
+     * @async
+     */
+    async drawImage({ model = "v3", prompt }) {
+        if (!["v1", "v2", "v2-beta", "v3", "lexica", "prodia"].some(ind => model == ind)) model = "prodia";
+        if (!prompt || prompt == undefined || prompt == null) throw new Error("Please specify a prompt!");
+        try {
+            const apiUrl = `https://hercai.onrender.com/${model}/text2image` + "?prompt=" + encodeURI(prompt);
+            const response = await fetch(apiUrl, {
+                method: 'GET',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+            });
+
+            if (!response.ok) {
+                throw new Error(`HTTP error! Status: ${response.status}`);
+            }
 
+            const data = await response.json();
+            return data;
+        } catch (err) {
+            throw new Error("Error: " + err.message);
+        }
+    }
 }
 
 module.exports = Hercai;
